controllers/storagecluster/storagecluster_controller_test.go

```diff
@@ -81,13 +82,13 @@ var mockStorageClusterWithArbiter = &api.StorageCluster{
 
 var mockCephCluster = &rookCephv1.CephCluster{
 	ObjectMeta: metav1.ObjectMeta{
-		Name:      generateNameForCephCluster(mockStorageCluster),
+		Name:      generateNameForCephCluster(mockStorageCluster.DeepCopy()),
```

Comment:
Even if you don't use DeepCopy here, it's fine. Because, generate function won't modify the object.

Reply:
yes, true. I wanted to make sure that we don't use mock variables directly regardless of their modification.